crates/ruff_python_formatter/tests/fixtures.rs

```diff
@@ -0,0 +1,187 @@
+use ruff_python_formatter::format_module;
+use similar::TextDiff;
+use std::fmt::{Formatter, Write};
+use std::fs;
+
+#[test]
+fn black_compatibility() {
+    insta::glob!(
+        "../resources",
+        "test/fixtures/black/**/*.py",
+        |input_path| {
```

Comment:
Could you move these into their own function? In my experience macro calls often break tooling inside of it

Reply:
It worked 😮